File: PruebaPacientes/PruebaPacientes/SignosYSintomas.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Modelo
{
    public class SignosYSintomas
    {
        public int Id                           { get; set; }

        public bool FiebreMayorA38              { get; set; }
        public bool Taquipnea_Disnea            { get; set; }
        public bool Diarrea                     { get; set; }
        public bool Artralgias                  { get; set; }
        public bool MalestarEnGenerl            { get; set; }
        public bool Tos                         { get; set; }
        public bool Vomitos                     { get; set; }
        public bool DolorDeGarganta             { get; set; }
        public bool InsuficienciaRespiratoria   { get; set; }
        public bool DolorAbdominal              { get; set; }
        public bool DolorToracico               { get; set; }
        public bool RechazoAlAlimento           { get; set; }
        public bool IrritabilidadConfusion      { get; set; }
        public bool Convulsiones                { get; set; }
        public bool Otros                       { get; set; }
        public string OtrosDetalle              { get; set; }
        //public int FormularioParaAnalisisId { get; set; }

        //public FormularioParaAnalisis FormularioParaAnalisis { get; set; }

    }
}

File: Frontend/api/models/SignosYSintomas.js
/**
 * SignosYSintomas.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {
    FiebreMayorA38:{
      type: 'boolean',
      defaultsTo: false,
    },
    Taquipnea_Disnea:{
      type: 'boolean',
      defaultsTo: false,
    },
    Diarrea:{
      type: 'boolean',
      defaultsTo: false
    },
    Artralgias:{
      type: 'boolean',
      defaultsTo: false
    },
    MalestarEnGenerl:{
      type: 'boolean',
      defaultsTo: false
    },
    Tos:{
      type: 'boolean',
      defaultsTo: false
    },
    Vomitos:{
      type: 'boolean',
      defaultsTo: false
    },
    DolorDeGarganta:{
      type: 'boolean',
      defaultsTo: false
    },
    InsuficienciaRespiratoria:{
      type: 'boolean',
      defaultsTo: false
    },
    DolorAbdominal:{
      type: 'boolean',
      defaultsTo: false
    },
    DolorToracico:{
      type: 'boolean',
      defaultsTo: false
    },
    RechazoAlAlimento:{
      type: 'boolean',
      defaultsTo: false
    },
    IrritabilidadConfusion:{
      type: 'boolean',
      defaultsTo: false
    },
    Convulsiones:{
      type: 'boolean',
      defaultsTo: false
    },
    Otros:{
      type: 'boolean',
      defaultsTo: false
    },
    OtrosDetalle:{
      type: 'string'
    },
    //Relacion
    owner:{
      model:'FormularioParaAnalisis',
      unique:true
    }
  },

};

